src/System.Diagnostics.Contracts/src/System.Diagnostics.Contracts.builds

```diff
@@ -4,9 +4,6 @@
   <ItemGroup>
     <Project Include="System.Diagnostics.Contracts.csproj" />
     <Project Include="System.Diagnostics.Contracts.csproj">
-      <TargetGroup>net46</TargetGroup>
```

Comment:
Any reason we didn't use the `Net46 facade is currently inbox for 4.0` comment here?